bridge/Mixer.cpp

```diff
@@ -376,7 +376,7 @@ void Mixer::allocateAudioBuffer(uint32_t ssrc)
     auto audioBuffer = std::make_unique<EngineMixer::AudioBuffer>();
     auto* rawAudioBuffer = audioBuffer.get();
     _audioBuffers.emplace(ssrc, std::move(audioBuffer));
-    _engine.post(utils::bind(&EngineMixer::addAudioBuffer, _engineMixer.get(), ssrc, rawAudioBuffer));
+    _engine.post([&]() { _engineMixer->addAudioBuffer(ssrc, rawAudioBuffer); });
```

Comment:
you are capturing everything by reference but `ssrc` and `rawAudioBuffer` are a stack values. When the lambda run they will be dangling references